### PR TITLE
Python 3 compatibility added

### PR DESCRIPTION
**Pull request by zserg**
The PR to make Elodie Python 2 and 3 compatible.

### CODE REVIEW

**Comment by jmathai** [comment]
@zserg Thanks for this. Looks like the exiftool site is down which is causing the tests to fail. However I get some other failures when I try to run the tests which are not related. Here is what I see.

Some of these are not a big deal and I'm okay to update the `flake8` and `pep8` commands accordingly (see the `.pre-commit` file in the root of the repo). But I'd like to get the tests passing before I merge; even if that means updating the syntax checking for some of the issues.

```
(elodie-py3) ➜  elodie-py3 git:(zserg-py3_1) python -V
Python 3.5.2
(elodie-py3) ➜  elodie-py3 git:(zserg-py3_1) ./.pre-commit
elodie/geolocation.py:6:1: E402 module level import not at top of file
elodie/geolocation.py:8:1: E402 module level import not at top of file
elodie/geolocation.py:9:1: E402 module level import not at top of file
elodie/geolocation.py:10:1: E402 module level import not at top of file
elodie/geolocation.py:10:1: F401 'fractions' imported but unused
elodie/geolocation.py:12:1: E402 module level import not at top of file
elodie/geolocation.py:13:1: E402 module level import not at top of file
elodie/geolocation.py:13:22: E401 multiple imports on one line
elodie/geolocation.py:15:1: E402 module level import not at top of file
elodie/geolocation.py:16:1: E402 module level import not at top of file
elodie/geolocation.py:80:80: E501 line too long (84 > 79 characters)
elodie/geolocation.py:151:43: E231 missing whitespace after ','
elodie/media/base.py:17:7: E111 indentation is not a multiple of four
elodie/media/base.py:19:7: E111 indentation is not a multiple of four
elodie/media/base.py:21:1: E302 expected 2 blank lines, found 1
elodie/media/media.py:12:1: F401 'builtins.object' imported but unused
elodie/media/photo.py:9:1: F401 'builtins.str' imported but unused
elodie/media/video.py:10:1: F401 'builtins.str' imported but unused
elodie/media/video.py:11:1: F401 'builtins.object' imported but unused
elodie/media/video.py:12:1: F401 'past.utils.old_div' imported but unused
```

File: elodie/external/pyexiftool.py [hunk 1]
```diff
@@ -403,11 +402,13 @@ def set_tags_batch(self, tags, filenames):
                             "an iterable of strings")
                 
         params = []
+        params_b = []
         for tag, value in tags.items():
             params.append(u'-%s=%s' % (tag, value))
             
         params.extend(filenames)
-        return self.execute(*params)
+        params_b = [x.encode('utf-8') for x in params]
```
Comment (jmathai) [inline elodie/external/pyexiftool.py]: Can we name this `params_utf8` for clarity?

File: elodie/tests/media/audio_test.py [hunk 2]
```diff
@@ -56,7 +57,7 @@ def test_get_date_taken():
     audio = Audio(helper.get_file('audio.m4a'))
     date_taken = audio.get_date_taken()
 
-    print '%r' % date_taken
+    print('%r' % date_taken)
```
Comment (jmathai) [inline elodie/tests/media/audio_test.py]: This must have been added by mistake. It can be removed.

**Comment by jmathai** [comment]
@zserg looked at the diff. Great work on this.

File: elodie/external/pyexiftool.py (hunk 1 above)
Comment (zserg) [inline elodie/external/pyexiftool.py]: forgot about style :) Fixed.

File: elodie/tests/media/audio_test.py (hunk 2 above)
Comment (zserg) [inline elodie/tests/media/audio_test.py]: done

**Comment by zserg** [comment]
@jmathai thanks for comments. All done.

**Pull request merged**